webapp/crashstats/settings/base.py

```diff
@@ -320,11 +320,6 @@ def filter(self, record):
 }
 
 
-# No need to load it because we don't do i18n in this project
-USE_I18N = False
-
-USE_L10N = False
```

Comment:
These are deprecated. I think that means Django always has localization enabled. We don't use that in Crash Stats, so I think this is probably fine and if not, someone will tell us and we can figure out what to do then.